orchagent/routeorch.h

```diff
@@ -54,6 +55,28 @@ struct NextHopObserverEntry
     list<Observer *> observers;
 };
 
+struct RouteBulkContext
+{
+    std::deque<sai_status_t>            object_statuses;    // Bulk statuses
+    NextHopGroupKey                     tmp_next_hop;       // Temporary next hop
```

Comment:
`tmp_next_hop `is only to handle the `addTempRoute `case, right?

Reply:
Yes, any concern?

---
In reply to: [410517262](https://github.com/Azure/sonic-swss/pull/1238#discussion_r410517262) [](ancestors = 410517262)

Reply:
Not really, it was for just clarification